instrumentation/http-tests/src/main/java/brave/test/http/ITHttpClient.java

```diff
@@ -310,10 +334,15 @@ protected Span checkReportsSpanOnTransportException() throws InterruptedExceptio
     return takeSpan();
   }
 
-  @Test public void addsErrorTagOnTransportException() throws Exception {
+  @Test public void errorTag_onTransportException() throws Exception {
```

Comment:
TODO: make this test as hard as the server ones (eventhough it will hurt). Right now, this only looks for existance of the "error" tag, which in the case of server, hid bugs.